charter.md

```diff
@@ -9,6 +9,7 @@ The intended work products of the SPACE RG include an evaluation of the impact o
 * Simulation, measurement, or testing techniques particularly well suited for aerospace networks.
 * Analysis of any impact on transport protocols as paths experience regular changes in characteristics like latency, jitter, and loss associated with nominal platform behavior.
 * Investigation of the challenges and benefits of tuning existing transport and application protocols in emerging aerospace network architectures.
+* Researching applicability of Traffic Engineering techniques for optimizing data flows in aerospace networks, considering unique challenges such as dynamic topologies, high latency, limited bandwidth, and the challenges of store-and-forward networking.
 * Exploration of management approaches that take into account aerospace-network-specifics. 
 * Conception of application models and interfaces considering human and robotic aerospace networks.
```

Comment:
So far, we have: 

1. SPACE Model (simulation, emulation)
2. SPACE Tuning (of existing protocols)
3. SPACE Traffic Engineering
4. SPACE Management/Operations
5. SPACE Application Models

We are missing crucial aspects such as Quality of Service, Reliability, and Security. These likely deserve a bullet on their own.

Reply:
There has been concern expressed about Routing in particular.

It's currently ruled out of this initial charter because we don't want 1,001 academic theses about how Starlink routing _might_ work.  The focus needs to be on real issues with the sincere hope that folks doing the work show up.

If we do get folks with real-world knowledge of the issues, then AIUI rechartering to broaden scope is a fairly simple matter in the IRTF (not exactly the case in the IETF, by comparison).